getting-started/pom.xml

```diff
@@ -71,6 +74,12 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.28.0</version>
```

Comment:
Can we extract this version to a variable?

Reply:
Yes, sure.